app/backend/resources/schema.edn

```diff
@@ -2123,5 +2123,13 @@
 
  {:db/ident :migration/subcontractor-role
   :txes [[{:db/doc "Not selectable in UI. Added automatically to subcontractor employees"
-           :db/ident :company-contract-employee.role/subcontractor}]]}]
+           :db/ident :company-contract-employee.role/subcontractor}]]}
+
+ {:db/ident :migration/change-cardinality-many-to-one-company-emails
+  :txes [[{:db/ident :company/emails :db/cardinality :db.cardinality/many}
```

Comment:
Is it not enough to just transact the `:db.cardinality/one`

Reply:
based on https://docs.datomic.com/cloud/schema/schema-change.html there 2 things to be done
- "When changing from a multi-valued attribute to a single-valued attribute, there must be at most a single value for every entity in the set of current assertions"
- transact the change of cardinality

I didn't include check for values as they are not a part of the schema

Reply:
>  there must be at most a single value for every entity in the set of current assertions; otherwise, the change will not be accepted and the transaction will fail.

The transaction will fail for dev. Maybe we should have a migration that just retracts the value first as there is nothing in the dev that would need to be saved?

Reply:
> there must be at most a single value for every entity in the set of current assertions; otherwise, the change will not be accepted and the transaction will fail.

So I'm assuming the migration fails in dev? Maybe we should retract all the emails and phones in another migration that's run first?

Reply:
New singular attributes :company/email and :company/phone-number were added and used instead of plural ones. Thas resolves the migration issue.

Reply:
fixed by using single email and phone-number